src/shiv/cli.py

```diff
@@ -163,10 +172,14 @@ def main(
                 if not Path(tmp_site_packages, "bin", console_script).exists():
                     sys.exit(NO_ENTRY_POINT.format(entry_point=console_script))
 
+        # Some projects need reproducible artifacts, so they can use SOURCE_DATE_EPOCH
+        # environment variable to specify the timestamps in the zipapp
+        timestamp = int(os.environ.get(SOURCE_DATE_EPOCH_ENV, time.time()))
```

Comment:
this can become

    timestamp = os.environ.get(SOURCE_DATE_EPOCH_ENV, 315554400) if reproducible else time.time()

where 315554400 is the epoch time for jan 1 1980

Reply:
:+1: